config/initializers/devise.rb

```diff
@@ -10,7 +10,7 @@
 
   # ==> Controller configuration
   # Configure the parent class to the devise controllers.
-  # config.parent_controller = 'DeviseController'
+  config.parent_controller = 'BaseDeviseController'
```

Comment:
This is necessary to use a different layout for the view

Reply:
Cool!